RELEASE_NOTES.md

```diff
@@ -4,7 +4,18 @@ The release notes are typically a highlighting subset of all changes made. For f
 
 Note that version 2.1 is the first version with release notes. Please see the commit history for older versions.
 
-## NEXT_RELEASE
+## 3.0a2
+
+### Improvements
+
+- Minor improvements to build system, documentation and clean-up of obsolete code for FMI 3.0. In general this release contains no significant new user-related improvements compared to 3.0a1.
+
+### Changes
+
+- Renamed several API methods for consistency between the different FMI versions. Primarily this concerns functions of names `fmi3_import_get_*` for retrieving variable properties, these are now named `fmi3_import_get_variable_*`, some examples are:
+    - `fmi3_import_get_variability` has been changed to `fmi3_import_get_variable_variability`.
+    - `fmi3_import_get_initial_unknowns_list` has been changed to `fmi3_import_get_variable_initial_unknowns_list`.
```

Comment:
Actually I'm not certain this is correct. `fmi3_import_get_variable_initial_unknowns_list` should not be renamed right? Look here:
`FMILIB_EXPORT fmi3_import_variable_list_t* fmi3_import_get_clocked_states_list(fmi3_import_t* fmu);`
`FMILIB_EXPORT fmi3_import_variable_list_t* fmi3_import_get_variable_initial_unknowns_list(fmi3_import_t* fmu);`

1. get_clocked_states was not renamed
2. `get_variable_*` indicates that we're retrieving something from the variable. Here we don't, we get it from the ModelDescription (i.e. first argument is `fmi3_import_t* fmu`)

Reply:
Fixed on master, just rebase and we're good